rocket-chat-android-widgets/src/main/res/layout/message_composer.xml

```diff
@@ -1,62 +1,57 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-        xmlns:app="http://schemas.android.com/apk/res-auto"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:orientation="vertical">
-
-    <chat.rocket.android.widget.DividerView
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content" />
-
-    <LinearLayout
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:gravity="bottom"
-            android:orientation="horizontal">
-
-        <ImageView
-                android:id="@+id/emoji_keyboard_toggle"
-                android:layout_width="32dp"
-                android:layout_height="32dp"
-                android:layout_margin="8dp" />
-
-        <chat.rocket.android.widget.message.ImageKeyboardEditText
-                android:id="@+id/editor"
-                android:layout_width="0px"
-                android:layout_height="wrap_content"
-                android:layout_gravity="center_vertical"
-                android:inputType="textCapSentences|textMultiLine"
-                android:background="@null"
-                android:layout_marginLeft="0dp"
-                android:layout_marginRight="0dp"
-                android:layout_marginTop="4dp"
-                android:layout_marginBottom="4dp"
-                android:padding="0dp"
-                android:layout_weight="1"
-                android:textSize="14sp"
-                android:hint="@string/message_composer_message_hint"
-                android:minLines="1"
-                android:maxLines="4" />
-
-        <FrameLayout
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content">
-
-            <ImageView
-                    android:id="@+id/btn_extras"
-                    android:layout_width="48dp"
-                    android:layout_height="48dp"
-                    android:padding="8dp"
-                    app:srcCompat="@drawable/ic_add_circle_outline_black_24dp" />
-
-            <ImageView
-                    android:id="@+id/btn_submit"
-                    android:layout_width="48dp"
-                    android:layout_height="48dp"
-                    android:padding="8dp"
-                    app:srcCompat="@drawable/ic_send_black_24dp" />
-
-        </FrameLayout>
-    </LinearLayout>
-</LinearLayout>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Can you reformat it following the new code style (https://github.com/RocketChat/Rocket.Chat.Android/blob/develop/CONDING_STYLE.md)?